applications/polkadot-runtime-releaser.md

```diff
@@ -0,0 +1,144 @@
+    - E.G. https://substrate.stackexchange.com/questions/11686/getting-error-rust-nightly-not-installed-when-starting-an-srtool-container-to-bu​a
+- When to notify the community to upgrade the node?
+  - Collator/Validator nodes should be upgraded first.
+  - RPC suppliers should be upgraded later.
+    - I saw some EVM RPCs require the runtime version equals to the node version.
+- How to host a wasm-override repository for community?
+  - Debug log WASM.
+    - We always use `sp-api/disable-logging` in the release build, so no more detail logs in the general runtime.
+    - We can enable the `sp-debug-derive/force-debug` in the debug build, so we can get more logs instead of `WASM stripped` (I think people often see this a lot while they are trying to get some debug log).
+  - EVM tracing WASM.
+    - For some EVM chains, they need to trace the EVM execution.
+
+PRR's goal is to streamline the process of releasing a new runtime for the Polkadot-SDK-based chain. It provides a standard way to release a new runtime, including the CI setup, runtime upgrade notification, and wasm-override repository hosting.
+
+### Ecosystem Fit
+
+For any Polkadot-SDK-based chain, they could use PRR to release their runtime. It's a tool that could help them to release the runtime in a more standard way.
+
+### Details(spec)
+
+For PRR's utilization, the chain team requires two repositories: one for their code and another for the release.
+
+Take `foo-network` as an example.
+They should have two repositories, `foo-network/foo` and `foo-network/foo-release`.
+
+`foo-network/foo` is the base Polkadot-SDK-based chain repository that everyone should have.
+
+`foo-network/foo-release` is a new repository that PRR requires.
+It will have 2 branches, `foo` and `bar` if they have 2 networks, `foo`(mainnet) and `bar`(canary network).
+It will host the override WASMs and the release under the corresponding network branch.
+
+For collator/validator nodes, they should monitor the `foo-network/foo` repository release as usual.
+For RPC suppliers or service node, they should monitor the `foo-network/foo-release` repository release, it's a more stable version. It will only be published once the on-chain runtime upgrade get confirmed.
+
+PRR will supply these components, which will all be hosted in one repository.
+- A GitHub Action to response the try-runtime. (Users can comment something like `/bot try-runtime <network>` under a release PR to trigger this action.)
+- A GitHub Action to build the release runtime.
+- A GitHub Action to notify release repository there is a new release that release repository can start preparing the override WASMs.
+- A GitHub Action and a tool to monitor the on-chain runtime version and compare it with the release runtime version to trigger the release on the release repository.
+- A tool to build and manage the override WASMs.
+
+Each component will have a very detailed guide to help the chain team to set up.
```

Comment:
Could you structure this a bit better, so it's apparent which actions will be supported and what kind of use case they will cover?

For example:

| GitHub Action | Trigger  | Specification  |
| -----: | ------ | -------------- |
| `try-runtime` | by commenting `/bot try-runtime <network>` on the PR | when executed, XYZ will happen |

Reply:
Updated.